Yoast/wordpress-seo
# Migrate inverted settings

## Summary

This PR can be summarized in the following changelog entry:

* Migrates the `hideeditbox-<post type>` and `hideeditbox-tax-<taxonomy>` settings to a saner `display-metabox-pt-<post type>` and `display-metabox-tax-<taxonomy>`.

## Test instructions

This PR can be tested by following these steps:

* Toggle the Yoast Metabox setting in `Search Appearance -> Content Types` for a particular post type (i.e. from true to false).
* Check that the Yoast SEO metabox isn't displayed for said post type when creating a new post / editing an existing one. Also check that the extra columns are not shown in the posts overview for that post type.
* Toggle the Yoast Metabox setting in `Search Appearance -> Taxonomies` for a particular taxonomy.
* Perform the same checks that you did for the post type.

## Quality assurance


Fixes #8992
